QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_002028	$The Fourth Research Material{nl}-{nl}Some poles unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
+QUEST_20150511_002029	$A wish would come true if we make a pot and put the wish in it. {nl}I understand that it's tradition, but for now, restoring Fedimian back to its former glory is more important than that.
+QUEST_20150511_002030	$I can't guarantee the safety of other holy relics or the keys so you better hurry.
+QUEST_20150511_002031	$I'm just listening! I'm not going to do what you say! {nl}I don't think wheat will grow here no matter what kind of magic you use. {nl}I need to look for my son's body, so you better keep it short!
+QUEST_20150511_002032	$The picture of the essence is drawn on the old surface.
+QUEST_20150511_002033	$The Eternal Adventure
+QUEST_20150714_002034	$Well, you're a new face. Did Julian send you?
+QUEST_20150714_002035	$Yeah. There's something weird about Goddess Laima. {nl}No one seems to know when she disappeared. To be honest nobody seems to really know whether or not she actually is a Goddess.
+QUEST_20150714_002036	$She's never appeared before. {nl}Although I've read somewhere that she does appear every now and then…
+QUEST_20150714_002037	$Even though it would be bad for my business, I wish I could just live peacefully rather than live in fear of monster attacks coming at any moment.
+QUEST_20150714_002038	$Welcome! {nl}While I can do basic repairs, I don't have the materials to create anything. {nl}Take a look at what I have.
+QUEST_20150714_002039	$When all is said and done, I don't lie about my prices. Everything I sell is priced honestly and accordingly.
+QUEST_20150714_002040	$Fire is a grace sent to us by the Goddess Gabija. {nl} You won't become a great Pyromancer if you don't acknowledge that.
+QUEST_20150714_002041	$The Goddesses may be all gone but my powers are still doing fine. {nl} I'm sure it's proof that Goddess Gabija is at least safe somewhere.
+QUEST_20150714_002042	$Killing enemies one by one from a distance is the beauty of the bow and arrow. {nl}It doesn't matter how many shots you have to take, the shots that actually hit {nl}are the ones that count.
+QUEST_20150714_002043	$Monk Lotze
+QUEST_20150714_002044	$I'll do whatever it takes to take back the monastery from the monsters.
+QUEST_20150714_002045	$I don't like those hunters. {nl}Outsiders must not interfere with the affairs of the monastery.
+QUEST_20150714_002046	$Monk Jidas
+QUEST_20150714_002047	$It would do us good if everyone would stopped being so stubborn. {nl}I'm sure the hunters will be a great help in retaking the monastery…
```

Comment:
> It would do us good if everyone would stopped being so stubborn. 

"stopped" -> "stop". Is the "would" here an expression of [politeness](http://www.englisch-hilfen.de/en/grammar/will_would_if_clauses.htm)? Otherwise it's not valid to use "would" here.